packages/itwin/grouping-mapping-widget/src/components/Mappings/hooks/useFetchExtractionStatus.ts

```diff
@@ -41,9 +42,9 @@ export const useFetchExtractionStatus = ({
           iconMessage: "No extraction found.",
         };
       } else {
-        if (latestJobStatus?.containsIssues) {
+        if (latestJobStatus?.state === ExtractionState.PartiallySucceeded || latestJobStatus?.state === ExtractionState.Failed) { // TODO: Check if this would entail `PartiallySucceeded` or `Failed`.
```

Comment:
Yes to this TODO.

Reply:
I confirmed with Mindaugas about the change, `containsIssues` was removed in favor of `PartiallySucceded`, `Failed` status is a rare one and it has to do with a bug in the agent or in the OData compiler, I think we should check for it too in case it happens.

Reply:
I though YES go check, lol!.